spec/plugins/browserGetSyncedConf.js

```diff
@@ -19,20 +19,26 @@ exports.config = {
   // Plugin patterns are relative to this directory.
   plugins: [{
     inline: {
-      onPageLoad: function() {
-        return q.delay(5000).then(function() {
-          protractor.ON_PAGE_LOAD = true;
+      onPageLoad: async function() {
+        return await new Promise(resolve => {
+          setTimeout(() => {
+            protractor.ON_PAGE_LOAD = true;
+            resolve();
+          }, 5000);
         });
       },
-      onPageStable: function() {
+      onPageStable: async function() {
```

Comment:
here `async` is redundant because function doesn't resolve promises inside

Reply:
It resolves the await on the return.

Reply:
yep :)